truffle-config.js

```diff
@@ -0,0 +1,74 @@
+const path = require("path");
```

Comment:
I deleted this by mistake :see_no_evil: Anyway added it back. Made zero changes, no need to review.